### PR TITLE
Routine maintenance 2023-07

### LINKED ISSUE #150: Dependency Dashboard
This issue lists Renovate updates and detected dependencies. Read the [Dependency Dashboard](https://docs.renovatebot.com/key-concepts/dashboard/) docs to learn more.

## Repository problems

These problems occurred while renovating this repository.

 - WARN: Using npm packages for Renovate presets is now deprecated. Please migrate to repository-based presets instead.

This repository currently has no open or pending branches.

## Detected dependencies

<details><summary>github-actions</summary>
<blockquote>

<details><summary>.github/workflows/publish.yml</summary>

 - `actions/checkout v3`
 - `actions/setup-node v3`

</details>

</blockquote>
</details>

<details><summary>npm</summary>
<blockquote>

<details><summary>package.json</summary>

 - `lodash ^4.17.21`
 - `util-deprecate ^1.0.2`
 - `@babel/cli ^7.22.9`
 - `@babel/core ^7.22.9`
 - `@launchpadlab/babel-preset ^2.1.1`
 - `@launchpadlab/eslint-config ^3.0.1`
 - `codeclimate-test-reporter ^0.5.1`
 - `documentation ^14.0.2`
 - `eslint ^8.46.0`
 - `husky ^8.0.0`
 - `jest ^29.6.2`
 - `redux ^4.2.1`
 - `redux-mock-store ^1.5.4`

</details>

</blockquote>
</details>

<details><summary>nvm</summary>
<blockquote>

<details><summary>.nvmrc</summary>

 - `node 18`

</details>

</blockquote>
</details>

<details><summary>travis</summary>
<blockquote>

<details><summary>.travis.yml</summary>

 - `node 18`

</details>

</blockquote>
</details>

---

- [ ] Check this box to trigger a request for Renovate to run again on this repository

### PR DESCRIPTION
**Pull request by chawes13**
- Resolves #150 
- Removes Symbol polyfill (deprecated by IE dying)
- Rename scripts (`prepublish` changes in later versions of npm)

### CODE REVIEW

File: test/middleware.test.js [hunk 1]
```diff
@@ -184,20 +185,21 @@ test('middleware responds after delay with provided data', () => {
...
-  
+
   // Activate the timer
   jest.runAllTimers()
   return pendingPromise
 })
 
 test('middleware rejects stubbed request after delay with error flag', () => {
   jest.useFakeTimers()
+  jest.spyOn(global, 'setTimeout')
```
Comment (chawes13) [inline test/middleware.test.js:202]: Resolves error received that `expect(setTimeout)` expects a spy

File: package.json [hunk 2]
```diff
@@ -36,22 +35,20 @@
     "lib/"
   ],
   "devDependencies": {
-    "@babel/cli": "^7.5.5",
-    "@babel/core": "^7.5.5",
-    "@launchpadlab/babel-preset": "^2.1.0",
-    "@launchpadlab/eslint-config": "^2.4.1",
-    "axios": "^0.18.0",
```
Comment (chawes13) [inline package.json:43]: We're mocking this package entirely, so we don't actually need to install it locally for anything

File: package.json [hunk 3]
```diff
@@ -36,22 +35,20 @@
...
-    "husky": "^3.0.4",
-    "jest": "^24.9.0",
-    "redux": "^4.0.4",
-    "redux-mock-store": "^1.2.3"
+    "@babel/cli": "^7.22.9",
+    "@babel/core": "^7.22.9",
+    "@launchpadlab/babel-preset": "^2.1.1",
+    "@launchpadlab/eslint-config": "^3.0.1",
+    "codeclimate-test-reporter": "^0.5.1",
+    "documentation": "^14.0.2",
```
Comment (chawes13) [inline package.json:43]: Only breaking change is bullet points go from `-` to `*`

File: package.json [hunk 4]
```diff
@@ -36,22 +35,20 @@
...
+    "eslint": "^8.46.0",
+    "husky": "^8.0.0",
+    "jest": "^29.6.2",
+    "redux": "^4.2.1",
+    "redux-mock-store": "^1.5.4"
   },
   "dependencies": {
-    "es6-symbol": "^3.1.1",
```
Comment (chawes13) [inline package.json:53]: This wasn't supported in IE, but that browser no longer needs support!

File: src/LP_API.js [hunk 5]
```diff
@@ -1,29 +1,27 @@
-import Symbol from 'es6-symbol'
```
Comment (bhennes2) [inline src/LP_API.js:1]: In the changelog for this version change I'd imagine we'll note that `IE6` support is now dropped?

**Review by bhennes2 — APPROVED** [review]
LGTM code-wise too! Added a question around changelog notes but that's non-blocking IMO.

File: src/LP_API.js (hunk 5 above)
Comment (chawes13) [reply src/LP_API.js:1]: Good point! I'll add that to the release notes. We typically don't maintain changelog files on client-side libs.

**Pull request merged**